Makefile

```diff
@@ -29,7 +29,7 @@ build-proto:
 		    ./src/agent-config/proto/hypertrace/agent/config/v1/config.proto
 
 .PHONY: build
-build: build-proto
+build:
 	python -m build
```

Comment:
re-building proto with newer versions of protoc will cause older versions of the python protobuf to not be able to load.